lib_xua/src/core/clocking/sw_pll_wrapper.h

```diff
@@ -0,0 +1,65 @@
+// Copyright 2024 XMOS LIMITED.
+// This Software is subject to the terms of the XMOS Public Licence: Version 1.
+
+#ifndef _SW_PLL_WRAPPPER_H_
+#define _SW_PLL_WRAPPPER_H_
+
+/* By default we use SW_PLL for Digital Rx configs running on xcore.ai */
+/* Note: Not yet implemented for Synchronous mode */
+#ifdef __XS3A__
```

Comment:
This should be done in xua_conf.h but has been discussed will happen in a different PR